src/output-tx.h

```diff
@@ -34,9 +34,11 @@ typedef int (*TxLogger)(ThreadVars *, void *thread_data, const Packet *, Flow *f
 /** packet logger condition function pointer type,
  *  must return true for packets that should be logged
  */
-//typedef int (*TxLogCondition)(ThreadVars *, const Packet *);
+typedef int (*TxLoggerCondition)(ThreadVars *, const Packet *, void *state);
```

Comment:
I think it's a bit strange that the Tx 'Condition' doesn't get a pointer to the, you know, tx ;)